include/llvm-dialects/Dialect/Dialect.td

```diff
@@ -169,7 +169,7 @@ def I64 : TgConstant<(IntegerType 64)>, Type;
 
 def PointerType : BuiltinType {
   let arguments = (args type:$self, AttrI32:$address_space);
-  let evaluate = "::llvm::PointerType::get($_context, $address_space)";
+  let evaluate = "($_context.supportsTypedPointers() ? ::llvm::PointerType::get(::llvm::Type::getInt8Ty($_context), $address_space) : ::llvm::PointerType::get($_context, $address_space))";
```

Comment:
Probably can just return an Int8PtrTy, since this will convert to opaque pointers in an opaque pointer scenario anyway?

Reply:
Yes, that should make all of this shorter :)